crypto/evp/e_aes_cbc_hmac_sha256.c

```diff
@@ -170,8 +173,13 @@ static size_t tls1_1_multi_block_encrypt(EVP_AES_HMAC_SHA256 *key,
 #  endif
 
     /* ask for IVs in bulk */
-    if (RAND_bytes((IVs = blocks[0].c), 16 * x4) <= 0)
+    IVs = blocks[0].c;
+    if (drbg != NULL) {
+        if (RAND_DRBG_generate(drbg, IVs, 16 * x4, 0, NULL, 0) <= 0)
```

Comment:
You could replace RAND_DRBG_generate() by RAND_DRBG_bytes(), now that #4752 has been merged.